deleteAndLoadSnapsets.sh

```diff
@@ -31,7 +32,24 @@ if [ -z "$COUCHDB_URL" ]; then
 fi
 
 log "Starting"
+log "Clear index: $CLEAR_INDEX"
+log "Static: $STATIC_DATA_DIR"
+log "Build: $BUILD_DATA_DIR"
+log "Node path: $NODE_PATH"
+log "Working directory: `pwd`"
 
+# Set up universal
+git clone https://github.com/GPII/universal.git
```

Comment:
I realize you are just moving what was there before (and I probably should have caught it when reviewing the change when Cindy made it ;)), but please add `--depth 1`. It will make the clone run faster, and will result in a (slightly) smaller Docker image.

Reply:
Sure.  Done.